src/app/products.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require_once '../../vendor/autoload.php';
+use Store\mySqlDbConnect as mySqlDbConnect;
+use Store\Store as Store;
+use Store\Category as Category;
+use Store\Product as Product;
+
+$mySqlCon = new mySqlDbConnect();
+$store = new Store($mySqlCon);
+$categories = $store->getCategories();
+$currentCategory = $store->getCurrentCategory($_GET['categoryId']);
```

Comment:
since using $_GET['categoryId'] twice - might be nice to make a variable for it and then use the variable